src/Pages/Register.js

```diff
@@ -3,12 +3,11 @@ import { Link } from 'react-router-dom';
 import axios from 'axios';
 
 function Registration() {
+  const [values, setInput] = useState('');
+  const [error, setError] = useState();
 
-    const [values, setInput] = useState('')
-    const [error, setError] = useState();
-
-    // func expression to post the data gathered in the inputfield
-    const submit = () => {
+  // func expression to post the data gathered in the inputfield
```

Comment:
I had to comment out this submit declaration because it was breaking the app.  I didn't change anything, just commented this out.